lib/metadata/wrapper.js

```diff
@@ -69,14 +69,15 @@ const metadata = {
 
     putObjectMD: (bucketName, objName, objVal, params, log, cb) => {
         log.debug('putting object in metdata');
-        client.putObject(bucketName, objName, objVal, params, log, err => {
-            if (err) {
-                log.warn('error from metadata', { implName, error: err });
-                return cb(err);
-            }
-            log.debug('object successfully put in metadata');
-            return cb(err);
-        });
+        client.putObject(bucketName, objName, objVal, params, log,
+            (err, data) => {
```

Comment:
@ttvi OK to expect metadata to send `versionId` in body of response to bucketclient (which stores in `data` param here for callback)?